lib/fetching/content-services/crowd-tangle-content-service.js

```diff
@@ -184,7 +185,20 @@ CrowdTangleContentService.prototype._parse = function(data, options) {
 	if (this.crowdtangle_lists[data.account.id]) {
 		metadata.ct_tag = this.crowdtangle_lists[data.account.id];
 	} else if (options.requestType == "SAVED_SEARCH") {
-		metadata.ct_tag = "Saved Search";
+		metadata.ct_tag = []
+		for(var saved_search_id in this.crowdtangle_saved_searches){
+			var saved_search = this.crowdtangle_saved_searches[saved_search_id];
+			var regex = new RegExp(saved_search.words.join("|"), 'gi');
```

Comment:
This will throw an error if saved_search.words contains characters like `(` that make the regex invalid. It's rare but we should at least catch the error here.